lldb/test/API/functionalities/postmortem/elf-core/TestLinuxCore.py

```diff
@@ -704,39 +704,105 @@ def test_riscv64_regs_gpr_fpr(self):
         process = target.LoadCore("linux-riscv64.gpr_fpr.core")
 
         values = {}
+        alias = {}
```

Comment:
I wonder if this would be neater if it was one dictionary where each entry was the expected value of the register, then any aliases it has.
```
{'ra': (0x00000000000101a4, "x1"}
```
Then it would also make sense to merge the two loops into:
```
for register in values:
  expected_value, alias = values[register]
  # check it reads as "register"
  if alias:
      # check it reads as "alias"
```